### DISCUSSION TITLE
Override styling and text?

### DISCUSSION THREAD

**Discussion by divyegd** [Adyen/adyen-react-native#447] [Q&A] [answered]
Hey guys,

Wondering if it's possible to override styling and text copy of the drop-in or react-native components?

**Comment by camil-adyen** [comment, marked as answer]
Hi @divyegd,

Please have a look at these files on [localization](https://github.com/Adyen/adyen-react-native/blob/develop/docs/Localization.md) and [customizations](https://github.com/Adyen/adyen-react-native/blob/develop/docs/Customization.md).
